roman_numeral.js

```diff
@@ -0,0 +1,55 @@
+/* Convert the given number into a roman numeral.*/
+
+/**
+ * @param {int} num
+ * @return {array}
+ * depending on the spec, returns an array with num details
+ */
+function getNumDetails(num){
+  var n = num;
+  var arr = [];
+  var spec = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1];
+  for(var i = 0; i < spec.length; i++){
+    arr.push(parseInt(n / spec[i]));
```

Comment:
Why `parseInt`? It's meant to convert strings into int. Here I think you have a float and you want the integer part: use `Math.floor` you can use our [website](https://www.google.com/search?q=integer+part+javascript). Coding is the skill of knowing the name of what you are looking for and copy pasting the answer.